### ISSUE TITLE
Any Update On This?

### ISSUE THREAD

**Issue by Siyanda-Mzam** [joltup/react-native-fetch-blob#66]
Hello,

Thanks so much for this awesome library. I wanted to find out if there is an update on [this issue](https://github.com/wkh237/react-native-fetch-blob/issues/272)?

Thanks a lot

Siya

**Comment by Traviskn** [comment]
I don't think we have an update on those particular issues, if you're still running into them would you be able to open new issues on this repo and provide code examples/repro steps to help us diagnose and fix?
